build/build-travis-wrapper.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+if [ "$#" -eq 1 ] && [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then
+  echo "Attemping to login to Docker Hub..."
+  docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
+  cd $TRAVIS_BUILD_DIR && ./build/build-container.sh $1 --push
```

Comment:
Given this is the only place that build-container.sh is called and it has the --push flag, why have you bothered with the flag?